modules/ROOT/pages/managing-users.adoc

```diff
@@ -89,7 +90,70 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Click *Create*.
 +
-. Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
+. Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, and then sign in through your identity provider to test the configuration.
+
+== Rotate a SAML Key
+
+As a security best practice, many organizations rotate the key used to encrypt SAML 2.0 assertions. With the Key Rotation feature, Anypoint Platform can generate a new key, or you can upload a public and private key pair (up to three keys). When you want to rotate a key, you can designate a new _primary key_ in Anypoint Platform and your IdP, and you can revoke the obsolete key to remove it from the rotation. 
```

Comment:
for consistent caps, don't cap Key Rotation here? If you choose to change, ping me directly for re-re-approval.